test/CadlRanchProjects/azure/core/lro/standard/src/Generated/StandardClient.cs

```diff
@@ -265,7 +266,7 @@ public virtual async Task<Operation<ExportedUser>> ExportAsync(WaitUntil waitUnt
 
             RequestContext context = FromCancellationToken(cancellationToken);
             Operation<BinaryData> response = await ExportAsync(waitUntil, name, format, context).ConfigureAwait(false);
-            return ProtocolOperationHelpers.Convert(response, ExportedUser.FromResponse, ClientDiagnostics, "StandardClient.Export");
+            return ProtocolOperationHelpers.Convert(response, FetchExportedUserFromResourceOperationStatusUserExportedUserError, ClientDiagnostics, "StandardClient.Export");
```

Comment:
Can we just name `FetchExportedUserFromResponse`?

Reply:
There is possibility that `ExportedUsers` could be results from different LRO with different raw responses. It would be better to name after `Fetch{ReturnType}From{RawResponseType}` to avoid naming conflicts.

Reply:
So the main difference is the `logicPath` among different raw responses. In that way, can we just write an common ResultFetch function `FetchExportedUserFromRequpons(Response response, string logicPath=null)`, in that way, we may not generate many Fetch methods.
But in order to support this, we need to update a lot in `ProtocolOperationHelpers`. It can be an refine in future.
Current it is ok to write multiple Fetch method, one for one Lro.